1.9/administration/gpu/aws-install.md

```diff
@@ -2,7 +2,7 @@
 post_title: Configure Your AWS Cluster
 nav_title: Configure Your AWS Cloud Cluster
 feature_maturity: experimental
-menu_order: 2
```

Comment:
I seem to remember @joel-hamill  saying that they mark everything with 1 here, and some external tool knows how to do the reordering when it gets published.